x/liquidfarming/keeper/auction.go

```diff
@@ -127,14 +127,20 @@ func (k Keeper) FinishRewardsAuction(ctx sdk.Context, auction types.RewardsAucti
 	poolCoinDenom := liquiditytypes.PoolCoinDenom(auction.PoolId)
 	farmingRewards := k.lpfarmKeeper.Rewards(ctx, liquidFarmReserveAddr, poolCoinDenom)
 	truncatedRewards, _ := farmingRewards.TruncateDecimal() // TODO: farm module may use sdk.DecCoins for sdk.Coins in the future
+	withdrawnRewardsReserveAddr := types.WithdrawnRewardsReserveAddress(auction.PoolId)
+	withdrawnRewardsReserves := k.bankKeeper.SpendableCoins(ctx, withdrawnRewardsReserveAddr)
+	totalRewards := truncatedRewards.Add(withdrawnRewardsReserves...)
 
 	winningBid, found := k.GetWinningBid(ctx, auction.Id, auction.PoolId)
 	if !found {
-		k.skipRewardsAuction(ctx, truncatedRewards, feeRate, auction)
+		k.skipRewardsAuction(ctx, totalRewards, feeRate, auction)
 	} else {
 		_, found = k.lpfarmKeeper.GetPosition(ctx, liquidFarmReserveAddr, poolCoinDenom)
 		if found {
-			_, fees, err := k.payoutRewards(ctx, auction.PoolId, feeRate, liquidFarmReserveAddr, poolCoinDenom, winningBid)
+			_, fees, err := k.payoutRewards(
+				ctx, auction.PoolId, poolCoinDenom, liquidFarmReserveAddr,
+				withdrawnRewardsReserveAddr, withdrawnRewardsReserves, winningBid, feeRate,
+			)
```

Comment:
This is not the comment on this line, but the comment for `if loop`. There is `if found`, but there is no `else` case. What if `GetPosition` returns `not found`? Of course, this would not happen. How about adding panic implementation in this case?

Reply:
Discussed this offline (`TestFinishRewardsAuction_NoOneFarmed` covers this case)